challenges/2021_cloud_base_height/04_eval_pytorch.ipynb

```diff
@@ -0,0 +1,1108 @@
+{
```

Comment:
<p>it might be nice to have a plot like this showing the info you showed earlier in a pie chart (percent too high, percent too low etc.) but shown on a geogrpahic plot like this one. So maybe a plot sdhowing where the cloub base is too high, another where it shows where it is too low etc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
I think it is interpretable from the plot, I added some text to explain this.